images/router/haproxy/reload-haproxy

```diff
@@ -12,8 +12,13 @@ if [ -f $pid_file ]; then
   old_pid=$(<$pid_file)
 fi
 
+# Assumes that the docker ip is the first entry in the hosts file.
+# This port is non-configurable via the router cmd, it is HAProxy specific.  It should
+# match the port used in the haproxy-config.template
+peer_name=$(awk 'NR==1 {print $1}' /etc/hosts):1937
```

Comment:
Agreed - let's resolve to not do any hacks to get our ip - the pod downward API should report that for us if we need it.

> On May 15, 2015, at 8:09 PM, ramr <notifications@github.com> wrote:
> 
> In images/router/haproxy/reload-haproxy:
> 
> > @@ -12,8 +12,13 @@ if [ -f $pid_file ]; then
> >    old_pid=$(<$pid_file)
> >  fi
> >  
> > +# Assumes that the docker ip is the first entry in the hosts file.
> > +# This port is non-configurable via the router cmd, it is HAProxy specific.  It should
> > +# match the port used in the haproxy-config.template
> > +peer_name=$(awk 'NR==1 {print $1}' /etc/hosts):1937
> Would it hurt to just bind on 0.0.0.0:1937 ?
> 
> —
> Reply to this email directly or view it on GitHub.
>